Parsers/ASimUserManagement/ASimUserManagementLinuxAuthpriv.yaml

```diff
@@ -0,0 +1,342 @@
+Parser:
```

Comment:
Please add this to union parsers as well.

Also, please add this parser's entry to EventVendor and Eventproduct values in Tester.csv for UserManagement

Reply:
Done